src/Domain/JapaneseParser.php

```diff
@@ -97,4 +97,29 @@ public function getParsedTokens(string $text, Language $lang) {
         return $tokens;
     }
 
+    public function getReading(string $text, Language $lang) {
```

Comment:
This method needs a unit test, if only to exercise the function, and to show what happens when non-Japanese text is parsed (e.g., some JP texts will include things like "NHK" -- maybe the reading should be left blank in that case).

Reply:
I am submitting a new version, I am more inclined to add tests once we are satisfied with the way things are.

Reply:
Got it 👍